doc/man/task-sync.5.in

```diff
@@ -85,6 +85,42 @@ Then configure Taskwarrior with:
 
     $ task config sync.gcp.bucket               <bucket-name>
 
+However you can bring your own service account credentials if your 
+`application-default` is already being used by some other application
+
+To begin, navigate to the "IAM and Admin" section in the Navigation Menu, then select "Roles."
+
+On the top menu bar within the "Roles" section, click "CREATE ROLE."
+Provide an appropriate name and description for the new role.
+
+Add permissions to your new role using the filter "Service:storage" (not the "Filter permissions by role" input box).
+Select the following permissions:
+
+    - storage.buckets.create    
+    - storage.buckets.get    
+    - storage.buckets.update
+    - storage.objects.create
+    - storage.objects.get
+    - storage.objects.list
+    - storage.objects.update
+
+ Create your new role.
+
+ On the left sidebar, navigate to "Service accounts."
+
+ On the top menu bar within the "Service accounts" section, click "CREATE SERVICE ACCOUNT."
+ Provide an appropriate name and description for the new service account.
+ Select the role you just created and complete the service account creation process.
+
+ Now, in the Service Account dashboard, click into the new service account and select "keys" on the top menu bar.
+ Click on "ADD KEY" to create a new key or upload an existing key (a JSON key).
```

Comment:
How would uploading an existing key work here? Maybe "or upload an existing key" should be omitted here?

Reply:
In the Keys tab of every service account, there is an option to upload our own keys below the create new key option 

<img width="846" alt="Screenshot 2024-01-22 at 6 48 45 AM" src="https://github.com/djmitche/taskwarrior/assets/65720968/5d8da311-897a-47e9-92d8-cf47e9414595">

The user can upload his/her existing keys from the local storage

<img width="685" alt="Screenshot 2024-01-22 at 6 48 51 AM" src="https://github.com/djmitche/taskwarrior/assets/65720968/da5b5c53-c426-4ab1-a225-a2fb4c4cf176">

Reply:
Yes, seems right. I'll change it to _Download/Create a new key_.
Plus, if someone is so experienced that they must upload a custom X509 key, then they must definitely know how to use it here!